Bottom: faa5d0c5e5f85814172ff8d206b448356b96d095
Top:    11d41f77ed44118198c2c08e6984d9a8191982a8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 19:04:09 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
